### PR TITLE
docs: add compose spec doc, fix podman compose doc

### LINKED ISSUE #1451: Docs: Compose Support
### Is your feature request related to a problem? Please describe

Documentation on the support of Compose with Podman Desktop

### Describe the solution you'd like

We should have documentation on the support of Compose provided by Podman Desktop
- [ ] How does it work
- [ ] How to enable/disable
- [ ] How does it work for a user who already have Compose on his environment
- [ ] How Compose will get updated

### PR DESCRIPTION
**Pull request by cdrage**
docs: add compose spec doc, fix podman compose doc

### What does this PR do?

* Removes redundant information / cleans up the document we had
  regarding Podman Compose
* Adds a Podman spec document that outlines how we currently support it

### Screenshot/screencast of this PR

See Netlify preview

### What issues does this PR fix or reference?

Closes https://github.com/containers/podman-desktop/issues/1451

### How to test this PR?

`yarn website:dev` or netlify preview

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

File: website/docs/compose/podman-compose.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: Podman Compose with Podman Desktop
+sidebar_position: 2
+---
+
+## Introduction
+
+[Podman Compose](https://github.com/containers/podman-compose#readme) is a community-driven tool that allows developers to use the [Compose Spec](https://compose-sec.io) to run multiple containers at the same time and facilitate communication between them. Podman Compose requires a [Compose YAML](https://compose-spec.io/) file with definitions about the containers that need to communicate.
```
Comment (benoitf) [inline website/docs/compose/podman-compose.md]:
```suggestion
[Podman Compose](https://github.com/containers/podman-compose#readme) is a community-driven tool that allows developers to use the [Compose Spec](https://compose-spec.io) to run multiple containers at the same time and facilitate communication between them. Podman Compose requires a [Compose YAML](https://compose-spec.io/) file with definitions about the containers that need to communicate.
```

File: website/docs/compose/podman-compose.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+title: Podman Compose with Podman Desktop
+sidebar_position: 2
+---
+
+## Introduction
+
+[Podman Compose](https://github.com/containers/podman-compose#readme) is a community-driven tool that allows developers to use the [Compose spec](https://compose-sec.io) to run many containers at the same time. Podman Compose requires a [Compose YAML](https://compose-spec.io/) file with definitions about the containers that need to communicate.
```
Comment (lstocchi) [inline website/docs/compose/podman-compose.md]: `https://compose-sec.io` -> `https://compose-spec.io`

**Review by lstocchi — CHANGES_REQUESTED** [review]

File: website/docs/compose/podman-compose.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+```sh
+podman-compose -f compose.yaml up -d
+```
+
+![img1](img/compose_doc_image_1.png)
+
+3. Manage Compose with Podman Desktop
+
+To confirm if the containers are running, open Podman Desktop and click on the ‘Containers’ option on the left side of the application. You will the containers bundled together under a Pod named `flask-redis (compose)` in this case. Here, the compose within the bracket indicates it has been generated from a Compose YAML file.
```
Comment (lstocchi) [inline website/docs/compose/podman-compose.md]: Missing verb, `you will ... the containers bundled`

**Review by lstocchi — APPROVED** [review]
LGTM, just a verb missing

**Comment by cdrage** [comment]
I'm going to just merge this as-is as we've had 2 reviews and this PR has been opened a for a month. We can make some edits / changes after when @themr0c is back

**Pull request merged**